DependencyInjection/Configuration.php

```diff
@@ -278,6 +278,18 @@ private function getOptionsConfig()
                     ->defaultFalse()
                     ->info('')
                 ->end()
+                ->scalarNode('max_navbar_notifications')
```

Comment:
thanks! shall we use integerNode() instead?

Reply:
Absolutly!

I'm still working on the possibility to test the [NavbarController](https://github.com/kevinpapst/AdminLTEBundle/blob/master/Controller/NavbarController.php). It seems that the only way is to create a micro-symfony-app and to get the response. Something like [this](https://github.com/facile-it/symfony-functional-testcase/tree/master/tests/App) and [this](https://github.com/liip/LiipFunctionalTestBundle/tree/master/Tests/App).

If you want, you can merge.... I think it will take me some other (long) time to create those tests :(